peripherals/spi/spi_v3.yaml

```diff
@@ -0,0 +1,281 @@
+      NoFault: [0, "No mode fault detected"]
+      Fault: [1, "Mode fault detected"]
+    TIFRE:
+      NoError: [0, "TI frame format error detected"]
+      Error: [1, "TI frame format error detected"]
+    CRCE:
+      Match: [0, "No CRC error detected"]
+      NoMatch: [1, "CRC error detected"]
+    OVR:
+      NoOverrun: [0, "No overrun occurred"]
+      Overrun: [1, "Overrun occurred"]
+    UDR:
+      NoUnderrun: [0, "No underrun occurred"]
+      Underrun: [1, "Underrun occurred"]
+    TXTF:
+      NotCompleted: [0, "Transmission buffer incomplete"]
+      Completed: [1, "Transmission buffer filled with at least one transfer"]
+    EOT:
+      NotCompleted: [0, "Transfer ongoing or not started"]
+      Completed: [1, "Transfer complete"]
+    DXP:
+      Unavailable: [0, "Duplex packet unavailable: no space for transmission and/or no data received"]
+      Available: [1, "Duplex packet available: space for transmission and data received"]
+    TXP:
+      Full: [0, "Tx buffer full"]
+      NotFull: [1, "Tx buffer not full"]
+    RXP:
+      Empty: [0, "Rx buffer empty"]
+      NotEmpty: [1, "Rx buffer not empty"]
+
+  IFCR:
+    "*C":
+      _write:
+        Clear: [1, "Clear interrupt flag"]
+
+  TXDR:
+    TXDR: [0, 4294967295]
+
+  RXDR:
+    _read:
+      RXDR:
+
+  CRCPOLY:
+    CRCPOLY: [0, 4294967295]
+
+  TXCRC:
+    _read:
+      TXCRC:
+
+  RXCRC:
+    _read:
+      RXCRC:
+
+  UDRDR:
+    UDRDR: [0, 4294967295]
+
+  CGFR:
+    MCKOE:
+      Disabled: [0, "Master clock output disabled"]
+      Enabled: [1, "Master clock output enabled"]
+    ODD:
+      Even: [0, "Real divider value is I2SDIV*2"]
+      Odd: [1, "Real divider value is I2SDIV*2 + 1"]
+    I2SDIV: [0, 255]
+    DATFMT:
+      RightAligned: [0, "The data inside RXDR and TXDR are right aligned"]
+      LeftAligned: [1, "The data inside RXDR and TXDR are left aligned"]
+    WSINV:
+      Disabled: [0, "Word select inversion disabled"]
+      Enabled: [1, "Word select inversion enabled"]
+    FIXCH:
+      NotFixed: [0, "The channel length in slave mode is different from 16 or 32 bits (CHLEN not taken into account)"]
+      Fixed: [1, "The channel length in slave mode is supposed to be 16 or 32 bits (according to CHLEN)"]
+    CKPOL:
+      SampleOnRising: [0, "Signals are sampled on rising and changed on falling clock edges"]
+      SampleOnFalling: [1, "Signals are sampled on falling and changed on rising clock edges"]
+    CHLEN:
+      Bits16: [0, "16 bit per channel"]
+      Bits32: [1, "32 bit per channel"]
+    DATLEN:
+      Bits16: [0, "16 bit data length"]
+      Bits24: [1, "16 bit data length"]
+      Bits32: [2, "16 bit data length"]
+    PCMSYNC:
+      Short: [0, "Short PCM frame synchronization"]
+      Long: [1, "Long PCM frame synchronization"]
+    I2SSTD:
+      Philips: [0, "I2S Philips standard"]
+      LeftAligned: [1, "MSB/left justified standard"]
+      RightAligned: [2, "LSB/right justified standard"]
+      Pcm: [3, "PCM standard"]
```

Comment:
Should be `PCM` unless there's a reason otherwise?

Reply:
ACK. I assume you mean capitalizing the variant, not shortening the doc.